### PR TITLE
added nimqt

### PR DESCRIPTION
**Pull request by faulander**
# nimtm - a wrapper for the QBittorrent Web-API
## QUICK EXAMPLE
```nim
import nimtm
let connection = initQb("http://127.0.0.1:8080", "admin", "adminadmin")
conn.login()
if conn.connected:
  var torrents = conn.getTorrents()
  for torrent in torrents:
    echo getTorrentFiles(torrent["hash"])
```

### CODE REVIEW

**Comment by Yardanico** [comment]
For a second I thought that this was a Qt wrapper by the package name

File: packages.json [hunk 1]
```diff
@@ -17463,5 +17463,19 @@
...
+  },
+    {
+    "name": "nimqt",
+    "url": "https://github.com/faulander/nimqt",
+    "method": "git",
+    "tags": [
+      "torrent",
+      "qbittorrent",
+      "api",
+      "wrapper",
```
Comment (narimiran) [inline packages.json:17475]: You need to remove this last comma (`,`), otherwise it is invalid JSON.

**Comment by faulander** [comment]
> For a second I thought that this was a Qt wrapper by the package name

I am sorry, but that's the appreviation they use for QBittorrent, i would have named it Qb otherwise. But i can rename it?

**Comment by narimiran** [comment]
> [qt is] the abbreviation they use for QBittorrent

You sure about that?

> But i can rename it?

I think lots of people would be confused to see that `qt` means `qbittorrent`, because they expect that it stands for... well... [Qt](https://en.wikipedia.org/wiki/Qt_(software)) :)

**Comment by faulander** [comment]
renamed nimqt to nimQBittorrent

**Pull request merged**